docs/operations/logging.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Tracing in SlateDB
+---
+
+# Tracing in SlateDB
+
+SlateDB uses the [`tracing`](https://github.com/tokio-rs/tracing) library for logging and diagnostics. This provides structured, contextual logging that helps debug and monitor your database operations.
+
+## Getting Started
+
+To enable tracing in your SlateDB application, you'll need to add the following dependencies to your `Cargo.toml`:
+
+```toml
+[dependencies]
+env_logger = "*"
+slatedb = "*"
+```
+
+## Basic Setup
+
+Here's a basic example showing how to consume tracing logs with `env_logger` and SlateDB:
+
+```rust
+uuse bytes::Bytes;
+use slatedb::config::DbOptions;
+use slatedb::db::Db;
+use slatedb::fail_parallel::FailPointRegistry;
+use slatedb::object_store::{memory::InMemory, ObjectStore};
+use std::sync::Arc;
+use log::{debug, error, log_enabled, warn, info, Level, trace};
+
+
+#[tokio::main]
+async fn main() {
+
+    // enable env logger
+    env_logger::init();
+
+    // test trace log
```

Comment:
These are all just showing how to use logging in general. Please remove as it's not relevant to SlateDB.